### PR TITLE
Some fixes for python sdk and aggregator support

### PR DESCRIPTION
**Pull request by davidiw**
* Add aggregator read ability
* Improve errors on resource not found
* Fixing CI/CD runs (work in progress)
* Fix linting issues that crept in due to poor CI/CD

### CODE REVIEW

File: ecosystem/python/sdk/aptos_sdk/bcs.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 MAX_U128 = 2**128 - 1
 MAX_U256 = 2**256 - 1
 
+
```
Comment (gregnazario) [inline ecosystem/python/sdk/aptos_sdk/bcs.py:22]: Why space :)

File: ecosystem/python/sdk/aptos_sdk/client.py [hunk 2]
```diff
@@ -632,3 +666,7 @@ def __init__(self, message, status_code):
         # Call the base class constructor with the parameters it needs
         super().__init__(message)
         self.status_code = status_code
+
+
+class ResourceNotFound(Exception):
+    """Errow thrown when the resource is not found"""
```
Comment (gregnazario) [inline ecosystem/python/sdk/aptos_sdk/client.py:672]: "Error". Also generally, it's better to describe the error.  E.g. the resource was not found.

File: ecosystem/python/sdk/aptos_sdk/ed25519.py [hunk 3]
```diff
@@ -83,8 +83,6 @@ def verify(self, data: bytes, signature: Signature) -> bool:
     @staticmethod
     def deserialize(deserializer: Deserializer) -> PublicKey:
         key = deserializer.to_bytes()
-        print(len(key))
```
Comment (gregnazario) [inline ecosystem/python/sdk/aptos_sdk/ed25519.py:86]: 🤡

**Review by gregnazario — APPROVED** [review]
Please cleanup the error message comment

File: ecosystem/python/sdk/aptos_sdk/client.py (hunk 2 above)
Comment (davidiw) [reply ecosystem/python/sdk/aptos_sdk/client.py:672]: rofl, copypasta from the previous error... though agree :)

File: ecosystem/python/sdk/aptos_sdk/bcs.py (hunk 1 above)
Comment (davidiw) [reply ecosystem/python/sdk/aptos_sdk/bcs.py:22]: python formatting rules :shrug:

File: ecosystem/python/sdk/aptos_sdk/ed25519.py (hunk 3 above)
Comment (davidiw) [reply ecosystem/python/sdk/aptos_sdk/ed25519.py:86]: :cricket:

File: ecosystem/python/sdk/aptos_sdk/client.py [hunk 4]
```diff
@@ -96,6 +96,40 @@ def get_table_item(
             raise ApiError(response.text, response.status_code)
         return response.json()
 
+    def aggregator_value(
```
Comment (banool) [inline ecosystem/python/sdk/aptos_sdk/client.py:99]: wow lol what a function, seems like something the API should support maybe. good for now though.

**Review by banool — APPROVED** [review]

File: ecosystem/python/sdk/aptos_sdk/client.py (hunk 4 above)
Comment (davidiw) [reply ecosystem/python/sdk/aptos_sdk/client.py:99]: I think it would probably make sense to make a Token API as well.

**Pull request merged**